edn/src/symbols.rs

```diff
@@ -98,12 +99,73 @@ impl Keyword {
 }
 
 impl NamespacedKeyword {
+    /// Creates a new `NamespacedKeyword`. When `name` starts with a `_`, it is considered
```

Comment:
nit: double newline after the first sentence.

Consider repeating the `is_reversed` notes, with an example, here.